### PR TITLE
added partitioned bulk insert benchmark

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: py/specs/partitioned_bulk_insert.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+insert_stmt = """insert into part_bulk_insert (id, value) values ({0}, '{1}')"""
+
+def random_string(char_length):
+    return ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(char_length))
+
+def setupInsertStmts(partitions):
+    setup_stmts = [create_table_stmt]
+    for i in range(partitions):
+        setup_stmts.append(insert_stmt.format(str(i), random_string(100)))
```
Comment (bogensberger) [inline py/specs/partitioned_bulk_insert.py]: why are you not using bulk args?

**Comment by chaudum** [comment]
I would not add a new redundant json file. Re-use the existing one!

File: py/specs/partitioned_bulk_insert.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    partitioned by (id)
+    with (number_of_replicas=0)
+"""
+
+insert_stmt = """insert into part_bulk_insert (id, value) values ({0}, '{1}')"""
+
+def random_string(char_length):
+    return ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(char_length))
+
+def setupPartitions(partitions):
```
Comment (chaudum) [inline py/specs/partitioned_bulk_insert.py]: naming: `setup_statements`

**Comment by chaudum** [comment]
ack

**Pull request merged**